config/channels/in-memory-channel/roles/controller-clusterrole.yaml

```diff
@@ -110,14 +110,7 @@ rules:
     resources:
       - "mutatingwebhookconfigurations"
       - "validatingwebhookconfigurations"
-    verbs: &everything
-      - "get"
-      - "list"
-      - "create"
-      - "update"
-      - "delete"
-      - "patch"
-      - "watch"
+    verbs: *everything
```

Comment:
The captured everything doesn't capture the `delete` verb.
```yaml
    verbs: &everything
      - get
      - list
      - watch
      - create
      - update
      - patch
```

Reply:
It isn't really "everything"

Reply:
/close